fleet_vehicle_inspection/security/fleet_vehicle_inspection.xml

```diff
@@ -24,4 +24,48 @@
         <field name="perm_unlink" eval="1"/>
     </record>
 
+    <!-- Record Rules -->
+
+    <record id="fleet_rule_inspection_visibility_user" model="ir.rule">
+        <field name="name">User can only see his/her inspection</field>
+        <field name="model_id" ref="model_fleet_vehicle_inspection"/>
+        <field name="groups" eval="[(4, ref('fleet.fleet_group_user'))]"/>
+        <field name="perm_read" eval="1"/>
+        <field name="perm_write" eval="0"/>
+        <field name="perm_create" eval="0"/>
+        <field name="perm_unlink" eval="0"/>
+        <field name="domain_force">[('vehicle_id.driver_id','=',user.partner_id.id)]</field>
+    </record>
+
+    <record id="fleet_rule_inspection_line_visibility_user" model="ir.rule">
+        <field name="name">User can only see his/her inspection</field>
+        <field name="model_id" ref="model_fleet_vehicle_inspection_line"/>
+        <field name="groups" eval="[(4, ref('fleet.fleet_group_user'))]"/>
+        <field name="perm_read" eval="1"/>
+        <field name="perm_write" eval="0"/>
+        <field name="perm_create" eval="0"/>
+        <field name="perm_unlink" eval="0"/>
+       <field name="domain_force">[('inspection_id.vehicle_id.driver_id','=',user.partner_id.id)]</field>
+    </record>
+
+    <record id="fleet_rule_inspection_visibility_manager" model="ir.rule">
```

Comment:
I think that this rule is not necessary because manager already has all permissions over this model. 
https://github.com/OCA/fleet/blob/c4051ac1c3630a2a35d1bbf333c699ae422ad86d/fleet_vehicle_inspection/security/fleet_vehicle_inspection.xml#L17

Reply:
I followed the same logic as the fleet module. And it is necessary because of the user rule.

https://github.com/odoo/odoo/blob/3d34d58388ab362bf8e69cc2e75adbdfda3331b2/addons/fleet/security/fleet_security.xml#L108

Reply:
In Fleet they are also applying that rule unnecessarily then hehe but its ok, it might be for greater security perhaps!